server/build.gradle

```diff
@@ -404,6 +404,20 @@ tasks.register("japicmp", me.champeau.gradle.japicmp.JapicmpTask) {
     failOnSourceIncompatibility = true
     annotationIncludes = ['@org.opensearch.common.annotation.PublicApi', '@org.opensearch.common.annotation.DeprecatedApi']
     annotationExcludes = ['@org.opensearch.common.annotation.InternalApi']
+    // Already broken APIs, excluding them for now
+    classExcludes = [
```

Comment:
@andrross @cwperks the damage has been already done, unless we are OK holding off the release, adding these exclusions

Reply:
Sure, we could do that, thanks @andrross

Reply:
@andrross mind approving please? thank you